### PR TITLE
(2309) Add admin "show" page showing decision data for organisation-level users

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

- Placeholder page for showing a single decision dataset

### Before

N/A

### After

localhost_3000_admin_decisions_e6fac99e-9ea2-4be8-bfe7-1c2dcf8bf6e6_cce0a88a-b688-486e-b4a9-52fccc68cee1_2022

### CODE REVIEW

**Review by Gweaton — APPROVED** [review]

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -311,6 +311,26 @@ Cypress.Commands.add('checkPublishNotBlocked', () => {
     });
 });
 
+Cypress.Commands.add('checkTable', (headings: string[], rows: string[][]) => {
```
[the remark this answers is not in the document]
Comment (bryangaledxw) [reply cypress/support/commands.ts:314]: This has persuaded me to go back and expand the seed data so we're showing multiple tables on this page

File: src/decisions/admin/decision.controller.spec.ts [hunk 2]
```diff
@@ -119,6 +141,73 @@ describe('DecisionsController', () => {
...
+      const result = await controller.show(
+        'example-profession-id',
+        'example-organisation-id',
+        '2017',
+        request,
+      );
+
+      expect(result).toEqual(expected);
+
+      expect(professionCheckSpy).toBeCalledWith(request, profession);
```
[the remark this answers is not in the document]
Comment (bryangaledxw) [reply src/decisions/admin/decision.controller.spec.ts:191]: Oh interesting! I'd wondered if there was any argument for one over the other

[the remark this answers is not in the document]
Comment (richpjames) [reply src/decisions/admin/decision.controller.spec.ts:191]: I would generally opt for `toHaveBeenCalledWith` as, in my experience, it is more commonly used and a bit more explicit

File: cypress/support/commands.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (bryangaledxw) [reply cypress/support/commands.ts:314]: I've expanded the seed data so we have multiple tables on this page, though I've gone for finding the table by caption, then using `.parent().within()`, which I feel like I prefer to adding an extra parameter to `checkTable`

**Pull request merged**